test/electionguard/test_election.cpp

```diff
@@ -32,3 +33,25 @@ TEST_CASE("Can serialize CiphertextElectionContext")
     CHECK(fromJson->getManifestHash()->toHex() == context->getManifestHash()->toHex());
     CHECK(fromBson->getManifestHash()->toHex() == context->getManifestHash()->toHex());
 }
+
+TEST_CASE("Assign ExtraData to CiphertextElectionContext")
+{
+    // Arrange
+    auto key = "ballot_base_uri";
+    auto value = "http://something.vote/";
+    unordered_map<string, string> extendedData({{key, value}});
+
+    // Act
+    auto context = CiphertextElectionContext::make(
+      3UL, 2UL, TWO_MOD_P().clone(), TWO_MOD_Q().clone(), TWO_MOD_Q().clone(), move(extendedData));
```

Comment:
Do we really want to do a move for the extended data or a clone?  The make function does a move under the hood already.